bigbluebutton-html5/imports/ui/components/plugins-engine/plugin-provided-state/component.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState, useContext } from 'react';
+import { PluginProvidedStateProps, PluginsProvidedStateMap, PluginObjects } from '../types';
+import * as PluginSdk from 'bigbluebutton-html-plugin-sdk';
+import { PluginsContext } from '../../components-data/plugin-context/context';
+import { ProvidedPlugins } from '../../components-data/plugin-context/types';
+
+const pluginsProvidedState: PluginsProvidedStateMap = {};
```

Comment:
As per [your comment](https://github.com/bigbluebutton/bigbluebutton/pull/17994/files#r1283171483), I propose that we rename the variable, so `pluginsProvidedState` turns into `pluginProvidedStateMap`.